Cubical/Codata/Conat/Properties.agda

```diff
@@ -148,10 +148,4 @@ module Bisimulation where
   path≡bisim = ua path≃bisim
 
   isProp≈ : ∀ {x y} → isProp (x ≈ y)
-  prove (isProp≈ p q i) = isProp≈′ (prove p) (prove q) i
-    where
-    isProp≈′ : ∀ {x y} → isProp (x ≈′ y)
-    isProp≈′ {zero} {zero} (con tt) (con tt) = refl
-    isProp≈′ {zero} {suc x} (con ()) (con ())
-    isProp≈′ {suc x} {zero} (con ()) (con ())
-    isProp≈′ {suc x} {suc y} (con p) (con q) = cong con (isProp≈ p q)
+  isProp≈ = subst isProp path≡bisim (isProp[Set≡] isSetConat)
```

Comment:
Ah! So the solution was to use `subst` and not `transport`.
I need to get a better intuition for one vs. the other.

Reply:
What is `subst` and `transport` called in the standard library?

It might be more intuitive for computer scientists if we call `transport` coercion (so `coe` or `coerce`).

Reply:
`subst` is called the same thing (Relation.Binary.PropositionalEquality). Interestingly I don't think we have `transport`... @gallais might correct me on this though?

Reply:
transport is dependent pattern matching on heterogeneous equality, afaik.